pkg/asset/machines/worker.go

```diff
@@ -106,10 +106,9 @@ func defaultGCPMachinePoolPlatform() gcptypes.MachinePool {
 	}
 }
 
-func defaultOpenStackMachinePoolPlatform(flavor string) openstacktypes.MachinePool {
+func defaultOpenStackMachinePoolPlatform() openstacktypes.MachinePool {
 	return openstacktypes.MachinePool{
-		FlavorName: flavor,
-		Zones:      []string{""},
+		Zones: []string{""},
 	}
```

Comment:
gratutious change between Zones: and [], but is just a nit.

Reply:
I believe go fmt would have made this change automatically.